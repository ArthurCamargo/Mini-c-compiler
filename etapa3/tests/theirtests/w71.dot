digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n1 -> n4;
n2;
n2 [label="c"];
n3;
n3 [label="2"];
n4;
n4 -> n5;
n4 -> n6;
n5;
n5 -> n7;
n5 -> n8;
n7;
n7 [label="a"];
n8;
n8 -> n9;
n8 -> n10;
n9;
n9 [label="c"];
n10;
n10 [label="c"];
n8 [label="+"];
n5 [label="[]"];
n6;
n6 [label="2"];
n4 [label="="];
n1 [label="="];
n0 [label="f"];

}