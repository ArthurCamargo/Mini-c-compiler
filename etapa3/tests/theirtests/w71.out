0x557808e4c3c0
0x557808e4c3c0, 0x557808e4c540
0x557808e4c540
0x557808e4c540, 0x557808e4c4a0
0x557808e4c540, 0x557808e4c4f0
0x557808e4c540, 0x557808e4c7d0
0x557808e4c4a0
0x557808e4c4a0 [label="c"];
0x557808e4c4f0
0x557808e4c4f0 [label="2"];
0x557808e4c7d0
0x557808e4c7d0, 0x557808e4c730
0x557808e4c7d0, 0x557808e4c780
0x557808e4c730
0x557808e4c730, 0x557808e4c5b0
0x557808e4c730, 0x557808e4c6e0
0x557808e4c5b0
0x557808e4c5b0 [label="a"];
0x557808e4c6e0
0x557808e4c6e0, 0x557808e4c620
0x557808e4c6e0, 0x557808e4c690
0x557808e4c620
0x557808e4c620 [label="c"];
0x557808e4c690
0x557808e4c690 [label="c"];
0x557808e4c6e0 [label="+"];
0x557808e4c730 [label="[]"];
0x557808e4c780
0x557808e4c780 [label="2"];
0x557808e4c7d0 [label="="];
0x557808e4c540 [label="="];
0x557808e4c3c0 [label="f"];
